### PR TITLE
chore(juno): create external app

### PR DESCRIPTION
**Pull request by ArtieReus**
# Summary

# Changes Made

- Change 1
- Change 2
- Change 3

# Related Issues

- Issue 1: [link to issue]

# Screenshots (if applicable)

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [ ] My changes generate no new warnings or errors.

### CODE REVIEW

File: packages/ui-components/eslint.config.mjs [hunk 1]
```diff
@@ -4,15 +4,19 @@
...
 export default [
   ...junoConfigs,
   {
     files: ["test/__mocks__/svgLib.js", "src/**/*.test.js"],
     languageOptions: { sourceType: "module" },
   },
-
   {
-    ignores: ["setupTests.js"],
+    rules: {
```
Comment (barsukov) [inline packages/ui-components/eslint.config.mjs:16]: What was the reason to apply it on ui-components lvl?

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "clean:cache": "turbo run clean:cache && rm -rf .turbo && rm -rf node_modules/.cache/turbo",
     "generate:package": "turbo gen workspace --type package --copy @cloudoperators/juno-package-template --name",
     "generate:app": "turbo gen workspace --type app --copy @cloudoperators/juno-app-template --name",
+    "generate:externalApp": "node scripts/generate-external-app.js",
```
Comment (barsukov) [inline package.json:17]: did you forget to commit this script? also why it is js?

**Review by barsukov — CHANGES_REQUESTED** [review]
Changes ;)

File: apps/template/index.html [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
+        display: inline;
+      }
+    </style>
   </head>
   <body>
     <div id="root"></div>
     <script type="module" src="/src/index.tsx"></script>
   </body>
-</html>
+</html>
```
Comment (andypf) [inline apps/template/index.html:36]: missing new line :(

File: packages/ui-components/eslint.config.mjs [hunk 4]
```diff
@@ -4,15 +4,19 @@
  */
 
 import junoConfigs from "@cloudoperators/juno-config/eslint/juno-typescript.mjs"
+import { RuleTester } from "eslint"
```
Comment (andypf) [inline packages/ui-components/eslint.config.mjs:7]: Why do we need the RuleTester here?

File: packages/ui-components/eslint.config.mjs (hunk 1 above)
Comment (andypf) [reply packages/ui-components/eslint.config.mjs:16]: We should consider adding those rules to the base configuration in the imported config package

**Review by andypf — COMMENTED** [review]
I added a few comments. It is difficult to understand what this PR is about since the script file is missing.

**Pull request closed without merging**